test/test_assert_that.py

```diff
@@ -141,3 +142,10 @@ def test_with_kwargs(self, assert_that, automation):
             assert_that(LIGHT).was_not.turned_off()
             automation.turn_off_light_with_transition(via_helper=True)
             assert_that(LIGHT).was.turned_off(transition=TRANSITION_DURATION)
+
```

Comment:
Test idea is to check that the INPUT_SELECT will be set to 'new_option' and maybe add another option later.

Reply:
Perfect. That test looks great. we ensure it was not set before, then act and check that it's been called right after. Awesome, we know that once this test pass, our implementation is correct 😃 (yours in this case 😁)

Reply:
Oh and actually . . . not _perfect_ 😅 When you call `select_option` you need to specify the `entity_id` as well as the option
```python
automation.select_option(INPUT_SELECT, 'new_option')
```

See: [Appdaemon Documentation - Select Option](https://appdaemon.readthedocs.io/en/latest/HASS_API_REFERENCE.html?highlight=select_option#appdaemon.plugins.hass.hassapi.Hass.select_option)

But that's not where your bug comes from. . . . or rather _came_ from. I just fixed it. You had forgotten . . . . . a comma 😁 (it happens to everyone ☺️)